test/bin/scenario.sh

```diff
@@ -197,22 +197,36 @@ launch_vm() {
     # FIXME: variable for vcpus?
     # FIXME: variable for memory?
     # FIXME: variable for ISO
-    if timeout "${VM_BOOT_TIMEOUT}" sudo virt-install \
+
+    local -r wait=$((VM_BOOT_TIMEOUT / 60))
+    local -r install_log=$(mktemp)
+    local -r install_log_dest="${WEB_SERVER_URL}/scenario-info/${SCENARIO}/vms/${vmname}/vm_install.log"
+
+    set +e
+    sudo virt-install \
          --noautoconsole \
          --name "${full_vmname}" \
          --vcpus 2 \
          --memory 4092 \
          --disk "pool=${VM_STORAGE_POOL},size=30" \
          --network network=default,model=virtio \
-         --events on_reboot=restart \
+         --noreboot \
          --location "${VM_DISK_DIR}/${boot_blueprint}.iso" \
-         --extra-args "inst.ks=${kickstart_url}" \
-         --wait; then
-        record_junit "${vmname}" "launch_vm" "OK"
+         --extra-args "inst.ks=${kickstart_url} console=ttyS0" \
+         --console "file,path=${install_log}" \
+         --wait "${wait}"
+    local -r install_result=$?
+    set -e    
+    sudo chown "$(whoami):$(whoami)" "${install_log}"
+    mv "${install_log}" "${install_log_dest}"
```

Comment:
How does this work if the `install_log_dest` is `${WEB_SERVER_URL}`

Reply:
I added this bit without testing :)